dotnet/src/SemanticKernel.Abstractions/Security/DefaultTrustService.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.Orchestration;
+using Microsoft.SemanticKernel.SkillDefinition;
+
+namespace Microsoft.SemanticKernel.Security;
+
+/// <summary>
+/// Default implementation of the trust service.
+///
+/// This is just a simple example implementation that will be used by default if no other is provided.
+///
+/// When set, throws an exception to stop execution when sensitive functions try to run with untrusted content.
+/// </summary>
+public class DefaultTrustService : ITrustService
```

Comment:
I think you can privatize the constructor here, and have 2 static instances with public static getters

```
public class TrustService 
{
   public static TrustService DefaultTrusted { get; } => new TrustService(true);
   public static TrustService DefaultUntrusted  { get; } => new TrustService(false);
}
```
Also: I prefer having the class name providing the defaults be the same as the interface, but without the "I" prefix (**TrustService.Default**)... makes it easy to guess, reach for when intellisensing. This is seen frequently throughout dotnet (for example, IEnumerable -> Enumerable.Empty<>() )

Reply:
This is a really nice comment. I am all in for this update, what do you think @dluc?